charbot/levels.py

```diff
@@ -149,13 +149,13 @@ async def proc_xp(self, message: discord.Message):
                 if gained + user["detailed_xp"][0] >= self._xp_function(user["level"]):
```

Comment:
```suggestion
                if gained + user["detailed_xp"][0] >= user["detailed_xp"][1]:
```
Its not required, but consider removing this function call because its technically unnecessary